<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="but_OnOff">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OnOff</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_Status">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>GetStatus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_UpdateList">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>UpdateList</string>
    </property>
   </widget>
   <widget class="QComboBox" name="list_Nodes">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>onoff</string>
    </property>
   </widget>
   <widget class="QListWidget" name="view_nodes">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>340</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
